app/src/ui/autocompletion/autocompleting-text-input.tsx

```diff
@@ -295,17 +286,28 @@ export abstract class AutocompletingTextInput<
     const autoCompleteText = autocompletionState.provider.getCompletionText(
       item
     )
+
+    const textWithAutoCompleteText =
+      originalText.substr(0, range.start - 1) + autoCompleteText + ' '
```

Comment:
This seems to be the current behavior on github.com and the old GitHub client. 
Just adding an extra space right after the inserted completion and position the caret there. Some other applications also do it this way. 
Slack seems to do something a bit less naive by actually looking at the next character and only inserting the extra space if the next character isn't one. 
Facebook will not add anything extra at all.

I chose to do it as the other GitHub clients to provide a unified experience.

Reply:
Yep, makes sense.